x-pack/solutions/security/plugins/security_solution/server/lib/siem_migrations/common/utils/audit.ts

```diff
@@ -131,9 +131,13 @@ export class SiemMigrationAuditLogger {
     });
   }
 
-  public async logUpdateMigration(params: { migrationId: string; error?: Error }): Promise<void> {
-    const { migrationId, error } = params;
-    const message = `User updated the SIEM migration with [id=${migrationId}]`;
+  public async logUpdateMigration(params: {
+    migrationId: string;
+    error?: Error;
+    field: 'name' | 'indexPattern';
+  }): Promise<void> {
+    const { migrationId, error, field } = params;
+    const message = `User updated the SIEM migration with [id=${migrationId}, field=${field}]`;
```

Comment:
When we change the "indexPattern", we are updating the rules documents, not the migration document. We need to call `logUpdateRules` instead.

Reply:
About logging the updated `field`, I don't see it being logged anywhere. I am not against adding it, but it's fine without it too. As you prefer